index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav>
+      <h2>Welcome</h2>
+      <img src="./images/menu.svg" alt="menu-icon" />
+    </nav>
```

Comment:
- [ ] Kindly be noticed that the tool bar should be in a `header` tag.
- [ ] The logo should be inside a link (`a` tag).
- [ ] The hamburger button should be inside a `nav` tag.

Reply:
Quick questions about the logo

- Is the "Welcome"  header the logo?
- Why should it be inside a link? Where should I link it to?